packages/stark-ui/src/modules/action-bar/components/_action-bar.component.scss

```diff
@@ -11,6 +11,13 @@
   overflow: hidden;
   transform-origin: bottom;
 
+  &.extended {
+    .stark-action-bar-action {
```

Comment:
What is this change about? There is just added code... no refactored code 🤔

Reply:
Due to the change applied in the template (`<label>` replaced by a `<span>` and `<div>` removed), it's needed to apply the `width: auto` when extended.
The `text-transform: none` should be applied in `.stark-action-bar-action.stark-action-bar-full .action-label` actually. I change this 😊